test/system/700-play.bats

```diff
@@ -386,3 +386,29 @@ status: {}
     run_podman rm -a -f
     run_podman rm -f -t0 myyaml
 }
+
+@test "podman kube play - hostport" {
+    HOST_PORT=$(random_free_port)
+    TESTDIR=$PODMAN_TMPDIR/testdir
```

Comment:
`$PODMAN_TMPDIR` is fresh and new for each `@test`; so I don't think you need `$TESTDIR`.